### PR TITLE
Migrate to message-queue-gonsumer v1

### PR DESCRIPTION
**Pull request by mchompalova**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -75,8 +76,14 @@ func main() {
...
 
-	logger.InitDefaultLogger(serviceName)
+	logLevel := app.String(cli.StringOpt{
+		Name:   "logLevel",
+		Value:  "INFO",
+		Desc:   "Logging level (DEBUG, INFO, WARN, ERROR)",
+		EnvVar: "LOG_LEVEL",
+	})
 
+	logger.InitDefaultLogger(serviceName)
```
Comment (ivan-p-nikolov) [inline main.go]: We should initialize logger v1 with the provided logLevel as well.

File: main.go [hunk 2]
```diff
@@ -115,7 +122,9 @@ func main() {
 		}
 
 		handler := video.NewVideoMapperHandler(producerConfig, httpClient)
-		messageConsumer := consumer.NewConsumer(consumerConfig, handler.OnMessage, httpClient)
+		logConf := logger2.KeyNamesConfig{KeyTime: "@time"}
+		l := logger2.NewUPPLogger(serviceName, *logLevel, logConf)
+		messageConsumer := consumer.NewConsumer(consumerConfig, handler.OnMessage, httpClient, l)
 		logger.Info(prettyPrintConfig(consumerConfig, producerConfig))
 
 		hc := video.NewHealthCheck(handler.GetProducer(), messageConsumer)
```
Comment (ivan-p-nikolov) [inline main.go]: Can we make `ch` on line 152 buffered?

File: main.go (hunk 1 above)
Comment (karsov) [reply main.go]: And move this line inside the "app.Action" block because it does not pick up command line settings here.

Comment (mchompalova) [reply main.go]: My bad. Didn't get it the first time.

**Pull request closed without merging**